mgr/ConsDefs.pm

```diff
@@ -173,8 +174,9 @@
     $CXXFLAGS = $CFLAGS = $LDFLAGS = $SOFLAGS = "$XMACHOPT";
 
     $CXXFLAGS      .= " -fPIC -w";
+$CXXFLAGS .= "-fdiagnostics-color=always -msse4.2 -std=c++11 -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -DNDEBUG";
```

Comment:
Not clear why we need the -msse4.2 option.  Is this for root 6, or are we just picking up a change for the heck of it?

Also... defines NDEBUG... which I am not convinced is the right thing to do here.  (Doesn't this break logic for creating debug vs nodebug builds?  Has this change to the CXXFLAGS been tested... beyond "it compiles, it runs, and plots can be made that sorta kinda look right")